packages/terra-core-docs/src/terra-dev-site/doc/search-field/example/SearchFieldBlock.jsx

```diff
@@ -1,18 +1,11 @@
 import React, { useState } from 'react';
 import SearchField from 'terra-search-field';
 
-const INVALID_MESSAGE = 'The default minimum search length is 2.';
-
 const SearchFieldBlock = () => {
-  const [searchText, setSearchText] = useState('');
-
-  const message = searchText.length >= 2 ? `Search text: ${searchText}` : INVALID_MESSAGE;
+  const [setSearchText] = useState('');
```

Comment:
Is this still needed if we are removing the `searchText` state variable?

Reply:
This is still needed for the search text updates to work and removing this will affect functionality of the examples

Reply:
Doesn't this just change `setSearchText` to be the state variable instead of the set function? Based on the useState doc: https://react.dev/reference/react/useState#returns

To me it seems like this is just passing the state variable `setSearchText` with value '' to the `onSearch` and `onInvalidSearch` props and the example would be equally functional by passing dummy functions to those props like `() => {}`.

Reply:
You are right, thanks for pointing this out. I also saw that the `onSearch` and `onInvalidSearch` are optional props which are not needed in the example after removing the minimum search length logic. So I have removed them [here](https://github.com/cerner/terra-core/pull/3849/commits/053b9ac552ec6d90f39535255fbc7a6914884a07).